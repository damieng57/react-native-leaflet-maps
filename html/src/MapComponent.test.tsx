import React from "react";
import { render } from "@testing-library/react";
import MapComponent from "./MapComponent";

describe("MapComponent", () => {
  test("it renders", () => {
    const { asFragment } = render(<MapComponent />);
    expect(asFragment()).toMatchSnapshot();
  });
});
