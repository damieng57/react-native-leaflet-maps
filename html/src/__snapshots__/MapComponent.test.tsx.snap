// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MapComponent it renders 1`] = `
<DocumentFragment>
  <div
    id="map-container"
    style="position: absolute; top: 0px; bottom: 0px; background-color: greenyellow; left: 0px; right: 0px;"
  />
</DocumentFragment>
`;
